tests/README.md

```diff
@@ -0,0 +1,22 @@
+# Testing process_pr.py
+
+## To replay the tests
+* Download [Framework.py](https://github.com/PyGithub/PyGithub/blob/v1.56/tests/Framework.py) from PyGithub repo (make sure to use this exact version) and place it in `tests` folder
+* Replace all `pool_size=self.pool_size` with `per_page=100` (a total of three replacements)
```

Comment:
@iarspider , is this really needed? why not override `github.Github` and set page size?

Reply:
I'm working on it